core/templates/dev/head/base.html

```diff
@@ -117,11 +117,11 @@
               <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
-                    <a ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
+                    <a ng-if="windowWidth < 768" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
```

Comment:
<=, otherwise you have a problem at exactly 768.

Actually a better way to implement this might be to have a JS function isWindowNarrow() that returns true or false. Then you just have to specify 768 in one place (and not sprinkle it throughout the HTML), which means that if we want to change that later, it's easy to do so.